comms.go

```diff
@@ -139,7 +140,7 @@ func (d *Device) ClearButtons() {
 
 // WriteColorToButton writes a specified color to the given button
 func (d *Device) WriteColorToButton(btnIndex int, colour color.Color) error {
-	img := getSolidColourImage(colour)
+	img := getSolidColourImage(colour, d.deviceType.imageSize.X)
```

Comment:
Pass in the imageSize from the device. In all current Stream Deck devices the imageSize X and Y sizes are the same.